privacy_security.qmd

```diff
@@ -1,81 +1,1064 @@
+
+* **Targeted Attacks:** in contrast to availability attacks, targetedattacks aim to compromise a small number of the testing samples.So the effect is localized to a limited number of classes, whilethe model maintains the same original level of accuracy on themajority of the classes. The targeted nature of the attackrequires the attacker to possess knowledge of the model's classes.It also makes detecting these attacks more challenging.
+
+* **Backdoor Attacks:** in these attacks, an adversary targetsspecific patterns in the data. The attacker introduces a backdoor(a malicious, hidden trigger or pattern) into the training data.For example, manipulating certain features in structured data ormanipulating a pattern of pixels at a fixed position. This causesthe model to associate the malicious pattern with specific labels.As a result, when the model encounters test samples that containthe malicious pattern, it makes false predictions.
+
+* **Subpopulation Attacks:** here attackers selectively choose tocompromise a subset of the testing samples, while maintainingaccuracy on the rest of the samples. You can think of theseattacks as a combination of availability and targeted attacks:performing availability attacks (performance degradation) withinthe scope of a targeted subset. Although subpopulation attacks mayseem very similar to targeted attacks, the two have cleardifferences:
+
+- **Scope:** while targeted attacks target a selected set of samples,subpopulation attacks target a general subpopulation with similarfeature representations. For example, in a targeted attack, anactor inserts manipulated images of a 'speed bump' warning sign(with carefully crafted perturbation or patterns), which causes anautonomous car to fail to recognize such sign and slow down. Onthe other hand, manipulating all samples of people with a Britishaccent so that a speech recognition model would misclassify aBritish person's speech is an example of a subpopulation attack.
+
+- **Knowledge:** while targeted attacks require a high degree offamiliarity with the data, subpopulation attacks require lessintimate knowledge in order to be effective.
+
+#### Case Study 1
+
+In 2017, researchers demonstrated a data poisoning attack against a popular toxicity classification model called Perspective [@hosseini2017deceiving]. This ML model is used to detect toxic comments online.
+
+The researchers added synthetically generated toxic comments with slight misspellings and grammatical errors to the model's training data. This slowly corrupted the model, causing it to misclassify increasing numbers of severely toxic inputs as non-toxic over time.
+
+After retraining on the poisoned data, the model's false negative rate increased from 1.4% to 27% - allowing extremely toxic comments to bypass detection. The researchers warned this stealthy data poisoning could enable the spread of hate speech, harassment, and abuse if deployed against real moderation systems.
```

Comment:
Wow this is super cool/effective/scary!

Reply:
Indeed, it is nuts! Some neat works out there.